README.md

```diff
@@ -168,6 +168,12 @@ When contributing, please make sure your code adheres to some basic coding guidl
   packages should list all packages. In rare cases, changes may omit the package name prefix.
 * All notable changes should be documented in the [Change Log](https://github.com/project-serum/anchor/blob/master/CHANGELOG.md).
 
+### Minimum version requirements
```

Comment:
Can you add this to the `docs/src/getting-started/installation.md` instead of the readme?

Reply:
@armaniferrante thanks for the quick tip! I've moved this section over to the installation.md file.

Reply:
Great thanks!